Analyzer/plugins/Analyzer.cc

```diff
@@ -899,20 +904,21 @@ void Analyzer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
       continue;
     }
     if (!isData) {
-      tuple->PrePreS_GenPtVsdRMinBckg->Fill(genColl[closestGenIndex].pt(), dRMinBckg);
+      tuple->BefPreS_GenPtVsdRMinBckg->Fill(genColl[closestGenIndex].pt(), dRMinBckg);
     }
-    if (!isData && dRMinBckg > 0.1 ) {
+    if (isBckg && dRMinBckg > 0.1 ) {
```

Comment:
Not sure it's good to do that way because we can be in the 'signal' case

Reply:
for signal there is another check which besides the distance also looks at the IDs, so this is fine like this